direct/src/gui/DirectButton.py

```diff
@@ -102,6 +127,23 @@ def commandFunc(self, event):
             # Pass any extra args to command
             self['command'](*self['extraArgs'])
 
+    def __repeat(self, state, event):
+        if state and not taskMgr.hasTaskNamed(self.uniqueName('__repeatFunc')):
+            task = self.addTask(self.__repeatFunc, self.uniqueName('__repeatFunc'), extraArgs=[event], appendTask=True)
```

Comment:
Rather than using a task by name, maybe it should just store the task as `self.__task` so that we can do `self.__task.remove()`?